spec/syntax.md

```diff
@@ -89,7 +89,8 @@ Attempting to parse a _message_ that is not _well-formed_ will result in a _Synt
 
 A _message_ is **_<dfn>valid</dfn>_** if it is _well-formed_ and
 **also** meets the additional content restrictions
-and semantic requirements about its structure defined below.
+and semantic requirements about its structure defined below for
+_declarations_, _matcher_ and _options_.
```

Comment:
Why is this addition necessary?

Reply:
It makes it clearer where in the doc to hunt for those "semantic requirements". IIRC we started with a single list that was pretty much the next paragraph after this, but the refactorings and later changes have moved them all over the place.